cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtilsTest.java

```diff
@@ -136,4 +141,39 @@ public void testParseExecutionProgressCheckIntervalMsWithValue() {
     EasyMock.verify(mockRequest);
     Assert.assertEquals(Long.valueOf(EXECUTION_PROGRESS_CHECK_INTERVAL_STRING), executionProgressCheckIntervalMs);
   }
+
+  @Test
+  public void testGetEndpoint() {
+    verifyGetEndpoint("/a/*");
+    verifyGetEndpoint(WebServerConfig.DEFAULT_WEBSERVER_API_URLPREFIX);
```

Comment:
I'd recommend dropping the test for the default -- i.e.
```
verifyGetEndpoint(WebServerConfig.DEFAULT_WEBSERVER_API_URLPREFIX);
```

and moving the content of the private function `verifyGetEndpoint` inside this test function as it is not used anywhere else and contains all the logic of this test.

Reply:
I have changed the implementation and the `verifyGetEndpoint` method is not used any more in test